modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+    Service:
+      Type:  
+  Tls:
+    Ca:
+Status:
+  Application Controller:      Running
+  Application Set Controller:  Unknown
+  Phase:                       Available # <1>
+  Redis:                       Running
+  Repo:                        Running
+  Server:                      Running
+  Sso:                         Unknown
+Events:                        <none>
+----
+<1> The status indicates that the cluster-scoped Argo CD instance is healthy and available.
++
+The {gitops-title} Operator creates the following default cluster roles and manages them:
++
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller` aggregated cluster role
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller-view`
+* `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` 
+
+. Verify that the Operator has created the default cluster roles and cluster role bindings for the Argo CD Application Controller and Argo CD server components by running the following commands:
++
+[source,terminal]
+----
+$ oc get ClusterRoles -l app.kubernetes.io/part-of=argocd
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                           CREATED AT
+example-spring-petclinic-argocd-application-controller         2024-08-14T08:20:58Z
+example-spring-petclinic-argocd-application-controller-admin   2024-08-14T09:08:38Z
+example-spring-petclinic-argocd-application-controller-view    2024-08-14T09:08:38Z
+example-spring-petclinic-argocd-server                         2024-08-14T08:20:59Z
+----
++
+[source,terminal]
+----
+$ oc get ClusterRoleBindings -l app.kubernetes.io/part-of=argocd
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                     ROLE                                                                 AGE
+example-spring-petclinic-argocd-application-controller   ClusterRole/example-spring-petclinic-argocd-application-controller   54m
+example-spring-petclinic-argocd-server                   ClusterRole/example-spring-petclinic-argocd-server                   54m
+----
++
+The cluster role bindings for the `view` and `admin` cluster roles are not created. This is because the `view` and `admin` cluster roles only add permissions to the aggregated cluster role and do not directly configure permissions to the Argo CD Application Controller.
++
+[TIP]
+====
+Alternatively, you can use the {OCP} web console to verify from the *Administrator* perspective. You can go to *User Management* -> *Roles* and *User Management* -> *RoleBindings*, respectively. You can search for the cluster roles and cluster role bindings that have the `app.kubernetes.io/part-of:argocd` label.
+====
+
+. Verify that the aggregated cluster role is created by checking the permissions of the aggregated cluster role,  `view`, and `admin` cluster roles by running the following command:
++
+[source,terminal]
+----
+$ oc get ClusterRole/<cluster_role_name> -o yaml
```

Comment:
```suggestion
$ oc get ClusterRole/<cluster_role_name> -o yaml # <1>
```
assuming this is a user-replaced value, we need a footnote

Reply:
Addressed